install/packages.rst

```diff
@@ -184,6 +184,22 @@ download-opkg
 Downloads an ipk file from the RobotPy and NI's online opkg repositories, along
 with the dependencies for the package.
 
+You can also use a `requirements.txt` file to specify which packages should be downloaded
```

Comment:
This sentence is lacking punctuation… you could join the next line to make it end with a colon.